### ISSUE TITLE
Training loss nan

### ISSUE THREAD

**Issue by txin96** [manoja328/tallyqacode#4]
Hi, I tried to run your code recently.
I converted .tsv file provided by [bottom-up-attention](https://github.com/peteanderson80/bottom-up-attention) to .h5 file and trained the RN_OG_embd on HowmanyQA.
Since the background features are not used in this model, I commented out the codes related to resnet152.h5.
But the training loss is always nan, all count predictions are 0, though gradient clip is set to 200.

Do you have any idea why the loss is always nan? 
And could you please provide the pre-trained model?
